package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "iaptic-client",
+  "version": "1.0.0",
+  "description": "",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "scripts": {
+    "test": "npx mocha --colors --require ts-node/register --extensions ts tests/*.test.ts tests/**/*.test.ts | bunyan -l error -o short --color",
+    "testw": "nodemon --ext ts --watch src/ --watch tests/ --watch config.ts --exec \"npm test\"",
+    "lint": "eslint src/ tests/ --ext .ts",
+    "trans": "tsc -b -v -i -w",
+    "build": "npx tsc",
+    "buildw": "npx tsc -w"
```

Comment:
`npx` should not be used in `package.json` file. In script present in `package.json`, only local packages can be used (which is a good thing, this way no global packages are used by mistake, this ensures portable and reproducible scripts executions).

In short, no `npx tsc` here, just `tsc`